src/screens/Settings/AboutSettings.tsx

```diff
@@ -66,6 +68,17 @@ export function AboutSettingsScreen({}: Props) {
           <SettingsList.PressableItem
             label={_(msg`Version ${appVersion}`)}
             accessibilityHint={_(msg`Copies build version to clipboard`)}
+            onLongPress={() => {
+              const newDevModeEnabled = !devModeEnabled
+              setDevModeEnabled(newDevModeEnabled)
+              Toast.show(
+                _(
+                  msg`Developer mode ${
+                    newDevModeEnabled ? 'enabled' : 'disabled'
+                  }`,
```

Comment:
Not sure the translation extraction is smart enough for this, probably best to simplify to this:
```suggestion
                newDevModeEnabled
                  ? _(msg`Developer mode enabled`)
                  : _(msg`Developer mode disabled`)
```